### ISSUE TITLE
Monaco Editor Release July 2017

### ISSUE THREAD

**Issue by rebornix** [Microsoft/monaco-editor#510]
We'll release Monaco Editor together with VS Code July release https://github.com/Microsoft/vscode/issues/30209 . This issue is used to track features/bugfixes/etc related to Monaco Editor.

## CSS
* CSS conflict with bootstrap https://github.com/Microsoft/monaco-editor/issues/496

## Accessibility
* Accessibility Help Dialog is not accessible on Edge/IE https://github.com/Microsoft/monaco-editor/issues/506

## Browser Compatibility
* IE11, Flexbox width inheritance https://github.com/Microsoft/monaco-editor/issues/505 
* IE11, remove property value https://github.com/Microsoft/monaco-editor/issues/513

## Comments
* Block comment enhancement https://github.com/Microsoft/vscode/pull/30818

**Comment by corbinu** [comment]
Any chance for #18 #40 #60 to get in the next iteration plan? Each of them on their own are the most 3 up voted issues (and two of the most commented) and they are really duplicates.

**Comment by rebornix** [comment]
@corbinu sorry I didn't put them on my task board at the beginning of this iteration, let me take a look and make it fit into our plan (not sure if I have room for them in Aug). Thanks.

**Comment by corbinu** [comment]
@rebornix Thanks so much. I don't think anybody will be mad if it takes a bit. I think we all just mostly want to see it on the schedule. Would totally say that if people think your biggest issue is they want to use your product more/more easily I think your doing pretty great ;)

**Comment by codepunkt** [comment]
@rebornix @corbinu yes. please!

**Comment by corbinu** [comment]
@rebornix Is there going to be a new release soon? Just wondering

**Comment by rebornix** [comment]
@corbinu yes we are going to have a new release this month

**Comment by corbinu** [comment]
Great any chance of getting the ES module support on the roadmap?

**Comment by raedle** [comment]
@rebornix This is really great news, and the community appreciates your hard work! Thank you, guys
